funnel/models/event.py

```diff
@@ -144,6 +144,18 @@ def remove_events(self, events):
             if event in self.events:
                 self.events.remove(event)
 
+    @classmethod
+    def checkin_list(cls, event):
+        participant_attendee_join = db.join(Participant, Attendee, Participant.id == Attendee.participant_id)
+        stmt = db.select([Participant.id, Participant.fullname, Participant.email, Participant.company, Participant.twitter, Participant.puk, Participant.key, Attendee.checked_in, Participant.badge_printed]).select_from(participant_attendee_join).where(Attendee.event_id == event.id).order_by(Participant.fullname)
+        return db.session.execute(stmt).fetchall()
+
+    @classmethod
+    def filter_by_ticket_type(cls, ticket_type):
+        participant_sync_ticket_join = db.join(Participant, SyncTicket, Participant.id == SyncTicket.participant_id)
+        stmt = db.select([Participant.id, Participant.fullname, Participant.email, Participant.company, Participant.twitter]).select_from(participant_sync_ticket_join).where(SyncTicket.ticket_type == ticket_type).distinct()
+        return db.session.execute(stmt).fetchall()
```

Comment:
Curious why you're stepping below the declarative ORM layer with this approach. This could simply be:

```python
return Participant.query.join(SyncTicket).filter(SyncTicket.ticket_type == ticket_type).all()
```

Unless the `DISTINCT` bit is doing something additional.